nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -267,6 +267,19 @@ in
         '';
       };
 
+      extraInitrd = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        example = /boot/extra_initrafms.gz;
```

Comment:
The example needs to be quoted, otherwise the evaluator will try to actually read it.